src/member.rs

```diff
@@ -0,0 +1,57 @@
+//! [`Member`]s and [`MemberId`]s are the building blocks of
+//! [`crate::problem::ProxyProblem`]s and [`crate::solution::ProxySolution`]s.
+use indexmap::map::IndexMap;
+
+use matchmaker::{Category, Student};
+
+use rocket::serde::Deserialize;
+use rocket_okapi::JsonSchema;
+
+/// An opaque [`String`] that uniquely keys a [`Member`] and identifies their
+/// [`MemberInfo`]`.preferences`.
+pub type MemberId = String;
+
+/// Alias for `matchmaker::Category`.
+pub type Proxy = Category;
+
+/// A `matchmaker::Student`.
+pub trait Member {
```

Comment:
Probably just a mistakenly object-oriented desire to alias `s/Student/Member/` and `s/Category/Proxy/` everywhere for clarity.  (@tellurideassociation does indeed deal with students, but this tool does not, so seeing `Student` all over the code, rather than a more abstract or theoretical term, is unnecessarily jarring.)

I think I'll leave it this way here.  But I appreciate the reality-check and will keep it in mind.